be/src/storage/lake/lake_primary_index.cpp

```diff
@@ -155,7 +159,6 @@ Status LakePrimaryIndex::_do_lake_load(Tablet* tablet, const TabletMetadata& met
         }
     }
     _tablet_id = tablet->id();
```

Comment:
`_tablet_id` is not set when using `LakeLocalPersistentIndex`.

Reply:
You are right, fix this.